cli/core/findStalePods.go

```diff
@@ -102,7 +104,14 @@ func isEigenpod(eth *ethclient.Client, chainId uint64, eigenpodAddress string) (
 		return false, nil
 	}
 
-	podOwnerShares, err := podMan.PodOwnerShares(nil, owner)
+	dmAddress, err := podMan.DelegationManager(nil)
+	if err != nil {
+		return false, fmt.Errorf("failed to reach delegationManager: %w", err)
+	}
+	dm, err := DelegationManager.NewDelegationManager(dmAddress, eth)
+
+	// TODO: are we looking for an aggregate sum here?
+	_, podOwnerShares, err := dm.GetDepositedShares(nil, owner)
```

Comment:
ditto comment here -- we're computing total TVL in `findStalePods` so we can understand whether the pod has slashed by more than N% of its beacon balance

@gpsanant - not sure if the calculation makes sense off deposited shares -- and presumably it's the sum across all strategies